### PR TITLE
NMS-13408: Document search panel

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/${JIRA-ISSUE-NUMBER}

### CODE REVIEW

File: docs/modules/operation/pages/provisioning/detectors/HostResourceSWRunDetector.adoc [hunk 1]
```diff
@@ -8,14 +8,14 @@ Most modern _SNMP agents_, e.g. _Net-SNMP agent_ and the _Microsoft SNMP service
 
 NOTE: This detector implements the configuration parameters inherited from the <<provisioning/detectors/SnmpDetector.adoc>>.
 
-==== Detector facts
+== Detector facts
```
Comment (mmahacek) [inline docs/modules/operation/pages/provisioning/detectors/HostResourceSWRunDetector.adoc:11]: These changes have already been pushed to foundation-2021.  Will that conflict with this commit?

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 2]
```diff
@@ -51,14 +51,14 @@ echo 'org.opennms.instance.id=MyNMS' > "$\{OPENNMS_HOME}/etc/opennms.properties.
 ----
 
 Update the Minion's instance ID accordingly to match the {page-component-title} instance.
-The same ID instance property should be present in `${MINION_HOME}/etc/custom.system.properties`.
+The same ID instance property should be present in `$\{MINION_HOME}/etc/custom.system.properties`.
```
Comment (mmahacek) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:54]: These changes have already been pushed to foundation-2021.  Will that conflict with this commit?

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 3]
```diff
@@ -40,7 +40,7 @@ Disabling Single topic must be configured on both Minion and OpenNMS.
 .Disable single topic on Minion
 [source, shell]
 ----
-echo 'single-topic=false' >> "$MINION_HOME/etc/org.opennms.core.ipc.rpc.kafka.cfg"
+echo 'single-topic=false' >> "$\{MINION_HOME}/etc/org.opennms.core.ipc.rpc.kafka.cfg"
```
Comment (mmahacek) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:43]:
```suggestion
echo 'single-topic=false' >> "$MINION_HOME/etc/org.opennms.core.ipc.rpc.kafka.cfg"
```

File: docs/modules/operation/pages/search/search.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+=== Enhanced Linkd topology search
+
+Find nodes with CDP/LLDP data that match the given search string.
+
+=== Category or Field asset information search
+Search for nodes that have assets associated with a particular category or field.
+The field search includes a list of fields and a text box to match a value.
+The field search is case-insensitive and inclusive.
+Use the underscore character as a single-character wildcard.
+Use the percent character as a multiple-character wildcard.
```
Comment (mmahacek) [inline docs/modules/operation/pages/search/search.adoc:86]: Probably should also include info about customizing that was part of a recent release.
Per `opennms.properties`:
```
# By default the central search shows only the found nodes and the corresponding matched field. This property
# allows you to add an additional line where you can use any meta-data DSL expression.
#
# This example will display the node's sys-contact and the associated department:
# org.opennms.netmgt.search.info=${node:sys-contact} (Department: ${asset:department})
#
# By default this property is not used and empty.
# org.opennms.netmgt.search.info=
```

```suggestion
Use the percent character as a multiple-character wildcard.

== Customizing search results

It is possible to customize the search results to show additional metadata in the search results pop up.
By creating a new file in `$OPENNMS_HOME/etc/opennms.properties.d/` and specifying a value for `org.opennms.netmgt.search.info`, you can pick additional metadata fileds to display.

This example will display the node's sys-contact and the associated department:

 org.opennms.netmgt.search.info=${node:sys-contact} (Department: ${asset:department})

**Review by mmahacek — COMMENTED** [review]
I think there may be a conflict with foundation-2021 due to some overlapping changes.

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc (hunk 2 above)
Comment (Bonrob2) [reply docs/modules/reference/pages/configuration/tuning-activemq.adoc:54]: @mmahacek. I'm not sure. I branched from the Release 28.x branch to work on this Jira, as the meta-data changes Christian did are a new feature. These issues showed up when I built locally, so wasn't sure how else to address them.

**Pull request closed without merging**